src/views/ActiveSpeakerView.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect } from "react";
+import { selectPeers, useHMSStore } from "@100mslive/hms-video-react";
+import { GridCenterView, GridSidePaneView } from "./components/gridView";
+import {
```

Comment:
move this above components imports. always import libs/deps first, then import local ones